xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd

```diff
@@ -826,6 +852,18 @@ Rail transport, Roads and Road transport
 			</xsd:restriction>
 		</xsd:simpleContent>
 	</xsd:complexType>
+	<xsd:complexType name="facilitiesRefs_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a collection of one or more FACILITies.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="oneToManyRelationshipStructure">
+				<xsd:sequence>
+					<xsd:element ref="FacilityRef" maxOccurs="unbounded"/>
```

Comment:
does not work:  there is no Facility element to refer with a FacilityRef !! see comment below in GroupOfFacilities

Reply:
done differently